subgraphs/rocket-pool/src/Constants/contractconstants.ts

```diff
@@ -0,0 +1,39 @@
+import { Address } from "@graphprotocol/graph-ts";
```

Comment:
should be named `contractConstants.ts` and the folder name should be `constants`

Reply:
Name and casing changed throughout project